db/version_edit.cc

```diff
@@ -55,6 +55,7 @@ enum CustomTag : uint32_t {
   // kMinLogNumberToKeep as part of a CustomTag as a hack. This should be
   // removed when manifest becomes forward-comptabile.
   kMinLogNumberToKeepHack = 3,
+  kHasGlobalSequence = 4,
```

Comment:
I think this one cannot be safely ignored, so it should be 66.
The comment "Mask for an identified tag from the future which can be safely ignored." might be ambiguous.

Reply:
OK